packages/collector/test/tracing/protocols/http/client/test.js

```diff
@@ -155,7 +155,15 @@ function registerTests(useHttps) {
               span => expect(span.ec).to.equal(1),
               span =>
                 expect(span.data.http.url).to.match(/ha-te-te-peh:\/\/999\.0\.0\.1(?:\/)?:not-a-port\/malformed-url/),
-              span => expect(span.data.http.error).to.match(/Protocol .* not supported./),
+              span => {
+                if (semver.gte(process.version, '16.2.0')) {
+                  expect(span.data.http.error).to.equal('Invalid URL');
+                } else if (semver.gte(process.version, '16.0.0')) {
+                  expect(span.data.http.error).to.match(/Invalid URL: /);
```

Comment:
Can't we just match `Invalid URL` for Node 16.* and we remove one if statement? Not a big deal though.
Maybe even ignore case, just in... case? (awful pun)

Reply:
Right, good call.